DomainContent/Hub/plates/ShatterPlateSpawner.js

```diff
@@ -42,16 +46,18 @@
 
     PlateSpawner.prototype = {
         preload: function(entityID) {
-            dishProperties = Entities.getEntityProperties(entityID, ['position', 'modelURL']);
+            position = Entities.getEntityProperties(entityID, 'position').position;
+            dishProperties.position = { x: position.x, y: position.y + POSITION_Y_OFFSET, z: position.z};
```

Comment:
could use Vec3.sum(position, {y: POSITION_Y_OFFSET}),   to shorten the code , but looks good as it is.